src/main/kotlin/lotto/domain/LottoCompany.kt

```diff
@@ -1,44 +1,41 @@
 package lotto.domain
 
-import lotto.constants.Messages
-import lotto.model.LottoResult
-import lotto.model.LottoTicket
-import lotto.model.Prize
+import lotto.constants.ErrorMessages
 
 /**
  * 당첨 번호를 갖고, 당첨 여부를 알려주는 클래스.
  * Created by Jaesungchi on 2022.05.25..
  */
-class LottoCompany(winningNumbers: List<Int>) {
-    val winningTicket: LottoTicket
-
+class LottoCompany(val winningTicket: LottoTicket, private val bonusNumber: Int) {
     init {
-        require(winningNumbers.size == LOTTO_COUNT_LIMITS) { Messages.Error.WINNING_NUMBER_IS_OVER_BASE }
-        winningTicket = LottoTicket(winningNumbers)
+        require(bonusNumber in LottoTicketFactory.LOTTO_NUMBER_RANGE) { ErrorMessages.NUMBER_IS_OVER_OR_UNDER_BASE }
+        require(!winningTicket.numbers.contains(bonusNumber)) { ErrorMessages.BONUS_IS_DUPLICATE_WITH_WINNINGS }
```

Comment:
```suggestion
        require(!winningTicket.contains(bonusNumber)) { ErrorMessages.BONUS_IS_DUPLICATE_WITH_WINNINGS }
```

winningTicket의 number를 꺼내서 bonusNumber를 포함되어있는지 확인하는게 아닌, winningTicket에게 number가 포함되어 있는지에 대해 물어보는건 어떨까요? 🤔